src/components/NewGalaxyModal.js

```diff
@@ -306,17 +316,72 @@ export const NewGalaxyModal = _.flow(
                 onlyInteger: true,
                 value: dataDiskSize,
                 style: { marginTop: '0.5rem', width: '5rem' },
+                disabled: !!currentDataDisk,
+                tooltip: !!currentDataDisk ?
+                  ['Persistent disk of size ', currentDataDisk.size, ' GB already exists and will be attached upon Galaxy creation'] :
+                  undefined,
                 onChange: v => setDataDiskSize(v)
               })
             ])
           ])
         ])
       ])
+  }
+
+  const renderDeleteDiskChoices = () => {
+    return div({ style: { ...styles.drawerContent, ...styles.warningView } }, [
+      h(TitleBar, {
+        style: styles.titleBar,
+        title: h(WarningTitle, ['Delete environment options']),
+        onDismiss,
+        onPrevious: () => {
+          setViewMode(undefined)
+          setDeleteDiskSelected(false)
+        }
+      }),
+      div({ style: { lineHeight: '1.5rem' } }, [
+        h(Fragment, [
+          h(RadioBlock, {
+            name: 'delete-persistent-disk',
```

Comment:
Should `name` be `keep-persistent-disk` or something?

Reply:
will do - will update `NewRuntimeModal` as well since it also only uses `'delete-persistent-disk'`